templates/zerver/help/send-a-group-of-people-a-private-message.md

```diff
@@ -1,76 +1,32 @@
-# Private Group Messages
-## There are three ways to send a private message to a group of people:
-### First Way:
-To send a new private group message,
-first click on the `New private message`
-button located at the bottom of your screen.
+# Send a group of people a private message
 
-![New private message](/static/images/help/private-message.png)
+You can easily send a new group private message through the compose box.
 
-The following box will be displayed:
+1. Click the **New private message** button located in the compose box the
+bottom of your screen.
 
-![New stream message](/static/images/help/private-box.png)
+2. Upon clicking on the the **New private message** button, the compose box
+will transform to include **one or more people...** and
+**Compose your message here...** inputs.
 
-In the **one or more people...** field,
-input the usernames or email addresses of the users
-you want to send a private group message to.
-Each username or email address should be separated by a comma (`,`).
-When you begin typing an email address or username,
-autocomplete suggestions will appear for individuals.
-The list of users in this field creates the group
-to which the private message will be sent.
+    ![Private message](/static/images/help/private-box.png)
 
-If you attempt to input a nonexistent username or email address,
-an error message will appear.
+3. In the **one or more people...** field, input all of the usernames or
+emails address of the users you want to send a group private message to.
+When you begin typing an email address or username, autocomplete suggestions
+will appear.
 
-### Second Way:
```

Comment:
The second way talks about Group PMs, which isn't related to creating a new private message, and the third way is documented in **Send someone a private message**.